docs/source/conf.py

```diff
@@ -121,7 +121,8 @@
 acquire_path = Path("../../submodules/acquire")
 
 autoapi_type = "python"
-autoapi_dirs = [acquire_path] + dissect_paths + flow_paths
+# autoapi_dirs = [acquire_path] + dissect_paths + flow_paths
+autoapi_dirs = [Path("../../submodules/dissect.util/dissect")]
```

Comment:
This looks like a debug lef-over

Reply:
Oh oops I thought I only committed a specific range.